helm_charts_updater/models.py

```
@@ -1,3 +1,11 @@
+"""Pydantic models for Helm chart metadata.
+
+This module defines the data models used to parse and validate
+Helm Chart.yaml files.
+"""
+
+from typing import Any
+from typing import Dict
```

Ruff (0.14.13)

`typing.Dict` is deprecated, use `dict` instead

(UP035)